adaptive/src/main/java/com/google/accompanist/adaptive/ExperimentalAdaptiveApi.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.adaptive
+
+@RequiresOptIn(message = "This APIs are experimental and may change in the future.")
+@Retention(AnnotationRetention.BINARY)
+annotation class ExperimentalAdaptiveApi
```

Comment:
I'm hesitant to add this at all because Accompanist doesn't make binary compat guarantees anyway and the whole repo should be considered experimental. Feedback is that this extra annotation is just confusing

Reply:
I see what you mean - I'll discuss with Jossi about potentially removing the annotation!

Reply:
I'm not opinionated here. We do have it on most other APIs, but we might not need it since it's a new API. Kristen, let's remove it for now?

Reply:
sounds good, just updated!